frontend/invites/invite.js

```diff
@@ -28,9 +28,9 @@ Invite.prototype.setType = function (typeOfInvite) {
 };
 
 Invite.prototype.areInstitutionsValid = function () {
-    const isInstitutionActive = this.institution.isStateOn('active');
-    const isRequestedInstActive = this.requested_institution ? this.requested_institution.isStateOn('active') : true;
-    return isInstitutionActive && isRequestedInstActive;
+    const isNotInstitutionInactive = ! this.institution.isStateOn('inactive');
```

Comment:
I think this bug only occurs because of this function. It was not considering the case in which the state is 'pending', so the requestType was set to INVALID_REQUEST, at selectDialogToShow.